### PR TITLE
Challenge review

### PR DESCRIPTION
**Pull request by jmticonap**
## Challenge review

### CODE REVIEW

**Review by otro34 — APPROVED** [review]

**Review by otro34 — CHANGES_REQUESTED** [review]
Complete requested changes.

**Comment by jmticonap** [comment]
Code is ok

**Review by jmticonap — COMMENTED** [review]
ok

File: src/main.ts
Comment (jmticonap) [inline src/main.ts:1]: This change allow the project work in railway

File: src/professor/dto/professor.dto.ts
Comment (jmticonap) [inline src/professor/dto/professor.dto.ts:1]: name was change

**Pull request merged**